common/introduction.adoc

```diff
@@ -83,7 +82,7 @@ Additionally, the following must be configured to use {product-title}:
 
 * The {product-title} appliance must already be installed and activated in your enterprise environment.
 * The SmartProxy must have visibility to the virtual machines and cloud instances that you want to control.
```

Comment:
Suggestion: The SmartProxy must have visibility **into** the virtual machines and cloud instances that you want to control.